app/controllers/dashboard_controller.rb

```diff
@@ -181,7 +181,7 @@ def guest
   def landing_page
     if params[:search]
       # Get matching Collections and Docsets
-      @search_results = Collection.search(params[:search]).unrestricted + DocumentSet.search(params[:search]).unrestricted
+      @search_results = search_results
```

Comment:
This line is repeated in new_landing_page. Moved it to a private function

Reply:
Would it be a bit cleaner/more functional if we passed `params[:search_results]` to the private function as an argument, so that it isn't as tightly coupled to the UI?

Reply:
Agree, will do this instead.